Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -911,6 +919,10 @@ void readDatalink(void){
             case SET_KIVALUES:
                 setKValues(GAIN_KI,(float*)(&cmd->data));
                 break;
+            case SET_GAINS:
+                int type = (int*) (&cmd->data);
```

Comment:
Sorry, when I created the merge request my push had failed and I didn't realize. I will push and create another merge request.